src/components/Landing.js

```diff
@@ -4,35 +4,11 @@ import '../assets/buttons.css';
 import './Landing.css';
 
 class Landing extends React.Component {
-  constructor ({ match }) {
-    super();
-    this.state = {
-      username: '',
-      period: '12month',
-      hideArtwork: false
-    };
 
-    this.handleUsernameChange = this.handleUsernameChange.bind(this);
-    this.handlePeriodChange = this.handlePeriodChange.bind(this);
-    this.handleHideArtworkChange = this.handleHideArtworkChange.bind(this);
-  }
-
-  handleUsernameChange(event) {
-    this.setState({
-      username: event.target.value
-    })
-  }
-
-  handlePeriodChange(event) {
-    this.setState({
-      period: event.target.value
-    })
-  }
-
-  handleHideArtworkChange(event) {
-    this.setState({
-      hideArtwork: event.target.checked
-    });
+  // Wrap change hanlder to keep parent code as independent as possible
```

Comment:
This comment describes your reason for writing this function as opposed to what you did in the beginning.  
I think you should describe what the function actually doing instead.

Reply:
the function wraps the change handler, and this is being done to keep the parent independent to changes....:man_shrugging: